migration/process-migration/src/main/java/io/camunda/migration/process/MigrationRunner.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.migration.process;
+
+import io.camunda.migration.api.Migrator;
+import org.springframework.stereotype.Component;
+
+@Component("process-migrator")
+public class MigrationRunner implements Migrator {
```

Comment:
shouldn't this be flagged with `@Profile("process-migration")` otherwise it's always running?

Reply:
Or alternatively be declared via a config class, so we can control it from there.

Reply:
My bad, I lacked awareness of `ProcessMigrationModuleConfiguration`, thanks @lenaschoenburg !